superset-frontend/src/components/Icons/index.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import AntdEnhancedIcons from './all/AntdEnhanced';
+import Trash from './all/Trash';
+import Alert from './all/Alert';
+import AlertSolid from './all/AlertSolid';
+import AlertSolidSmall from './all/AlertSolidSmall';
+import Binoculars from './all/Binoculars';
+import Bolt from './all/Bolt';
+import BoltSmall from './all/BoltSmall';
+import BoltSmallRun from './all/BoltSmallRun';
+
```

Comment:
I think this PR is looking AMAZING, but there's one thing I'm trying to weigh (and I'm curious what you think). 

Rather than having a whole bunch of small icon files, would it be easier to grow/maintain the icon set if we just made this file longer, doing all the VG imports and `<Icon ...>` composition in one place? ¯\\\_(ツ)_/¯ 

I think what you have done here is very clean and proper... I'm just wondering if the other way would make life easier. Kind of a nitpick, but now's the time to think about it I suppose.

Reply:
I second this, I'd prefer not to touch 3 files (svg, component file, and this file) when adding a new svg icon

Reply:
@rusackas @nytai I tend to agree as well. I have made some changes and now the index imports, enhances, and exports everything back. This makes the file a bit lengthy but it keeps all the functionalities discussed plus what you want here. This is a work-in-progress and I still need to finish re-exporting all the existing icons, but I'd appreciate your feedback while I move forward with this.

Reply:
This is looking pretty good. It may get messy eventually, but that might just mean we need to clean up our icons. I think it's pretty clear what changes are needed to add a new icon from glancing at this file.